bin/copy_sphinxgallery.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Script to do a local install of sphinx-gallery
+rm -rf tmp sphinxgallery
+
+easy_install -Zeab tmp sphinx-gallery
+
+cp -vru tmp/sphinx-gallery/sphinxgallery/ .
+
+git add sphinxgallery
```

Comment:
I remembered that too. It's just that I would expect this kind of script to just get the newest sphinx-gallery without messing with my git status. Maybe you just want to test the latest sphinx-gallery version without any intent to commit the changes. 

Also it's not like missing a file is a something that we don't realise right away. IIRC it was caught during the review and it would have been caught next time someone tried to generate the doc anyway.